parser.gemspec

```diff
@@ -20,27 +21,26 @@ Gem::Specification.new do |spec|
     'source_code_uri' => "https://github.com/whitequark/parser/tree/v#{spec.version}"
   }
 
-  spec.files         = `git ls-files`.split + %w(
-                          lib/parser/lexer.rb
-                          lib/parser/ruby18.rb
-                          lib/parser/ruby19.rb
-                          lib/parser/ruby20.rb
-                          lib/parser/ruby21.rb
-                          lib/parser/ruby22.rb
-                          lib/parser/ruby23.rb
-                          lib/parser/ruby24.rb
-                          lib/parser/ruby25.rb
-                          lib/parser/ruby26.rb
-                          lib/parser/ruby27.rb
-                          lib/parser/ruby30.rb
-                          lib/parser/macruby.rb
-                          lib/parser/rubymotion.rb
-                       ) - %w(
-                          .gitignore
-                       )
+  spec.files         = Rake::FileList['{bin,lib}/**/*'].exclude(*File.read('.gitignore').split)
```

Comment:
This is a bit hard to follow. Also it forces `.gitignore` to have entries that are valid for `FileList` (this I why you had to change it I suppose)

Does it work if you do this?

```ruby
Dir['bin/*'] + Dir['lib/**/*.rb'] + 'parser.gemspec'
```

You can check it by doing

```
$ gem uninstall parser
$ gem uninstall ast
$ rake build
$ gem install pkg/parser-2.7.1.4.gem
$ ruby-parse -e '1 + 1'
```

Reply:
also `spec/test_files` seems to be optional, and we don't want to include test files into release build, could you remove this assignment too, please?

Reply:
Perfect, anything that's pure Ruby works for us! Just not `git` 😄 
I'll fix this right away! 😄

Reply:
Does it work for you now?